serialization/src/PlumedForceProxy.cpp

```diff
@@ -41,16 +41,18 @@ PlumedForceProxy::PlumedForceProxy() : SerializationProxy("PlumedForce") {
 }
 
 void PlumedForceProxy::serialize(const void* object, SerializationNode& node) const {
-    node.setIntProperty("version", 2);
+    node.setIntProperty("version", 3);
     const PlumedForce& force = *reinterpret_cast<const PlumedForce*>(object);
     node.setStringProperty("script", force.getScript());
+    node.setDoubleProperty("temperature", force.getTemperature());
     node.setBoolProperty("restart", force.getRestart());
 }
 
 void* PlumedForceProxy::deserialize(const SerializationNode& node) const {
-    if (node.getIntProperty("version") != 2)
+    if (node.getIntProperty("version") != 3)
```

Comment:
This breaks backward compatibility.  It should accept any version number from 1-3 inclusive.  If `version < 3`, it just leaves the temperature set to the default value.  The `restart` flag should work the same way - sorry I missed that when reviewing the earlier PR.

Reply:
Done!